src/main/frontend/handler/editor.cljs

```diff
@@ -1775,7 +1779,8 @@
   (when (and input
              (state/get-editor-action)
              (not (wrapped-by? input page-ref/left-brackets page-ref/right-brackets)))
-    (when (get-search-q)
+    ;; TODO Why do we need get-search-q to be truthy here? Seems to work fine without it.
```

Comment:
If we remove the call here, then it's a unused function and we can remove it.

Need to know what the `get-search-q` stand for @tiensonqin

Reply:
Yeah, definitely. Just leaving it as it is for now so my temporary code comments make sense.

Reply:
I did some dig-in. 
The function return `true` equivalent when text is focused (no matter inside / outside of `[[]]`). It returns `nil` when the editor is not focused.

Reply:
Hrm. This seems like a very convoluted way to do that. And making sure there's an active editor should be covered by `(when-let [id (state/get-edit-input-id)]`, yeah? Or is there  case when that would give an ID, but the text isn't focused, that I need to consider?

Reply:
I think this function used to do something but now effectively does the much simpler thing you mean, and isn't necessary anymore; so I think we should just remove it and the place where we check it. Sound good?

Reply:
Misread the code a little; I think `input` being truthy guarantees that this will operate only on the input you're interacting with (it passes in either the input you clicked on, or the input you keyup'd in), so I think this is fine to remove.

Reply:
Agree. Can just remove it

Reply:
Cool, this is no longer WIP then and ready for final review :)

Reply:
@phoenixeliot Thank you! I will have a full check on Monday :)

Reply:
Sorry for the late response. Yes, it could be deleted now.